package.json

```diff
@@ -27,31 +40,32 @@
   },
   "homepage": "https://github.com/readmeio/fetch-har#readme",
   "dependencies": {
-    "@readme/data-urls": "^1.0.1",
-    "@types/har-format": "^1.2.12",
-    "readable-stream": "^3.6.0",
-    "undici": "^5.24.0"
+    "@readme/data-urls": "^3.0.0",
+    "@types/har-format": "^1.2.13",
+    "undici": "^5.25.1"
```

Comment:
Do we need to bring this in if we're just using whatever native fetch() is present in the environment?

Reply:
@kanadgupta @domharrington We could move this to `optionalDependencies` but yeah this is only really needed for server environments.

Reply:
Annoyingly, `File` only became a global in Node v20: https://nodejs.org/docs/latest/api/globals.html#class-file otherwise we could just use that.